### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by narendran-kannan**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (harish-sethuraman) [inline .github/ISSUE_TEMPLATE/feature_request.md]: This one alone shall we change to "Why do we need this feature "?
also include "Does this feature require installation of new package ?"

**Review by harish-sethuraman — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (narendran-kannan) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Okay

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Have you tested your addition is not affecting any existing code and working properly?
+
+
+# Checklist:
+
+Please read and confirm all this checklist, we'll merge only if your PR get checks in all the items below. If Not applicable, please remove that particular item.
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
```
Comment (harish-sethuraman) [inline .github/pull_request_template.md:28]: Do we really need comments @naresh3599  😄

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue) Type the issue ID below
```
Comment (harish-sethuraman) [inline .github/pull_request_template.md]: Also include add Issue link

**Review by harish-sethuraman — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 2 above)
Comment (narendran-kannan) [reply .github/pull_request_template.md:28]: Yes, da. It'll be good to go.

File: .github/pull_request_template.md (hunk 3 above)
Comment (narendran-kannan) [reply .github/pull_request_template.md]: Okay!

**Review by harish-sethuraman — APPROVED** [review]

**Pull request merged**